lib/reverb/Reverb.cc

```diff
@@ -449,12 +449,13 @@ void MixxxPlateX2::processBuffer(const sample_t* in, sample_t* out, const uint f
 	DSP::FPTruncateMode _truncate;
 
 	// loop through the buffer, processing each sample
+	// note (timrae):treat the effect as SEND type instead of INSERT type for smoother parameter changes
```

Comment:
It may be helpful to expand on this comment and note that the effects system might obsolete this in the future.